### PR TITLE
Fontawesome fix 0

### PR DESCRIPTION
**Pull request by adrientremblay**
# Assignee Section

## Description
Reverts back to fontawesome 4.7.0 to fix broken control icons.  All icons are working in this branch.  Local test for PR 272 passed.

closes #[Insert your issue number here]

The approval from all software team leads is necessary before merging.

# Reviewer Section

Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.

- [x] Local Test Performed Successfully
- [x] [General Integration Test](https://docs.google.com/document/d/1ug0CpA1cIzURP8DDFSvCt2CEJJSwJ6Ta6B1LG_hYk6I/edit) Performed Successfully

### CODE REVIEW

**Comment by PeterGhimself** [comment]
Added a few extra fixes.
Seems to work locally, gonna test it again with the rover tmrw morning. :crossed_fingers:

File: robot/basestation/templates/elements/controls/MultiCameraFeed.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    </div>
+
+    <div class="col">
+        <img id="front-camera-feed" src="" class="w-100" alt="Arm Camera Stream Offline" onerror="this.onerror=null;this.src='../static/images/stream-offline.jpg';"/>
+        <span>
+            <strong>Front Rover Stream</strong>
+        </span>
+        <div class="form-check checkbox-slider--a checkbox-slider-lg">
+            <label>
+                <input id="front-stream-btn" type="checkbox">
```
Comment (MartensCedric) [inline robot/basestation/templates/elements/controls/MultiCameraFeed.html]: wtf is this

Comment (MartensCedric) [reply robot/basestation/templates/elements/controls/MultiCameraFeed.html]: this file is not used anymore @PeterGhimself

Comment (MartensCedric) [reply robot/basestation/templates/elements/controls/MultiCameraFeed.html]: not sure how it got added but wtv ill delete

**Pull request merged**